ocaml/xapi/xapi_pool_update.ml

```diff
@@ -238,9 +278,63 @@ let pool_update_yum_repo_handler (req: Request.t) s _ =
     )
 
 let introduce ~__context ~vdi =
-  let vdi_name = Db.VDI.get_name_label ~__context ~self:vdi in
-  let update_info = load_update_info_from_xml ~__context vdi_name in
-  create_update_record ~__context update_info
+  let r = Ref.make () in
+  let _ = Db.Pool_update.create ~__context
+    ~ref:r
+    ~uuid:(Uuid.string_of_uuid (Uuid.make_uuid ()))
+    ~name_label:"deadbeef"
+    ~name_description:"deadbeef"
+    ~installation_size:0L
+    ~key:""
+    ~after_apply_guidance:[]
+    ~vdi:vdi
+  in
+  let verify update_info =
+    let update_xml_path = String.concat "/" [Xapi_globs.host_update_dir; update_info.uuid; "vdi"; "update.xml"] in
+    let repomd_xml_path = String.concat "/" [Xapi_globs.host_update_dir; update_info.uuid; "vdi"; "repodata"; "repomd.xml"] in
```

Comment:
It's a bit harder to use `Filename.concat` here, so I suggest you keep this as it is!